src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -1,4 +1,6 @@
 import _ from '../../../../platform/utilities/data';
+// TODO: Write a merge function
```

Comment:
I didn't want to write two data util functions in one PR, so I stuck with using lodash for merge.